facts/Vagrantfile

```diff
@@ -342,7 +342,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.box = 'generic/fedora36'
     host.vm.synced_folder '.', '/vagrant'
     # there's no official support for fedora 36, so we need facter from Fedora
-    host.vm.provision 'shell', inline: 'dnf -y install facter ruby ruby-devel wget make gcc net-tools'
+    host.vm.provision 'shell', inline: 'dnf -y install facter ruby ruby-devel wget make gcc net-tools augeas'
```

Comment:
Since this affects the actual fact generation, it would be nice for it to be in the shared script instead of set per-machine, but that probably needs attention on all the machines, not just this one.

Reply:
yes. I think we should clean up the Vagrantfile later on